src/cbmc_viewer/markup_link.py

```diff
@@ -31,10 +31,7 @@ def path_to_file(dst, src):
 
     path = os.path.relpath(dst, os.path.dirname(src))
     if dst != os.path.normpath(os.path.join(os.path.dirname(src), path)):
-        raise UserWarning(
-            "{} != {}".format(dst,
-                              os.path.normpath(os.path.join(os.path.dirname(src), path)))
-        )
+        raise UserWarning(f"{dst} != {os.path.normpath(os.path.join(os.path.dirname(src), path))}")
```

Comment:
very nit: I'd personally factor that long expression into a variable

actually, maybe not very nit, the same expression is used in the preceding if

Reply:
+1

Reply:
Done.